docs/components/connectors/out-of-the-box-connectors/slack.md

```diff
@@ -89,7 +89,8 @@ To post a message, take the following steps:
 2. Set **Channel/User Name** to either the **channel** or **user** you want to send the message to.
    1. A **channel** is specified by a unique identifier starting with a `#` (for example, `#myChannel`).
    2. A **user** is specified by a username starting with an `@` symbol (for example, `@myUser`).
-3. Select a **Message type**.
+3. (Optional) A thread can be specified, to start a thread from a specific message, `ts` in the response can be used (see [here](#post-message))
```

Comment:
thread => should be bold I guess? **thread**

Reply:
Yes it should, changing it right now